ISL.Providers.Notifications.GovUkNotifyIntercept/Services/Foundations/Notifications/NotificationService.cs

```diff
@@ -35,8 +35,8 @@ public ValueTask<string> SendEmailAsync(
             await ValidateInterceptingEmailAsync(interceptEmail);
```

Comment:
Think we should validate for a valid email address from config (and the same for the other two methods.  We need to enure what we substitute with is valid.)